### ISSUE TITLE
Error when trying to close the process avp.exe (Kaspersky)

### ISSUE THREAD

**Issue by An0nX** [Yaxser/Backstab#11]
Tried all the methods, nothing works

**Comment by L1B0** [comment]
Yeah, that's because avp uses ObRegisterCallbacks to register a function, which will monitor and prevent its process not be terminated.
